src/client/Microsoft.Identity.Client/MsalException.cs

```diff
@@ -42,6 +42,11 @@ public class MsalException : Exception
         /// </summary>
         public const string BrokerErrorCode = "BrokerErrorCode";
 
+        /// <summary>
+        /// An <see cref="AdditionalExceptionData"/> property key, available when using desktop brokers.
```

Comment:
Yes. I exposed it on AuthenticationResult as well. You can review commit by commit for more granular functionality.

Reply:
sorry should have been clear that the request was just to change the xml comment. tested the flow with the dev app and I do see the telemetry returned on success as well

Reply:
but I guess it is ok since this is in [MsalException.cs](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/4307/files/af51387737fff6df2f083c2efc0a66faa4e21c35#diff-0149c194ca9a8c938bc5b4f234f0be30e5f6210f7a622477d753cbee66776fb6)